includes/class-sensei-modules.php

```diff
@@ -1113,11 +1120,11 @@ class="wrap <?php echo esc_attr( $this->order_page_slug ); ?>">
 								if ( isset( $_GET['course_id'] ) ) {
 									$course_id = intval( $_GET['course_id'] );
 									if ( $course_id > 0 ) {
-										$modules = $this->get_course_modules( $course_id );
+										$modules = Sensei_Course_Structure::instance( $course_id )->get_modules();
 										$modules = $this->append_teacher_name_to_module( $modules, array( 'module' ), array() );
 										if ( $modules ) {
 
-											$order = $this->get_course_module_order( $course_id );
+											$order = wp_list_pluck( $modules, 'term_id' );
```

Comment:
When we get the list it already comes ordered. So I changed it to don't call again the `get_course_module_order`, and trust on the list order.